src/lib/server/services/content.spec.ts

```diff
@@ -149,4 +231,22 @@ describe.concurrent('content suite', () => {
         },
         generateStatContext
     );
+
+    testWithDb(
+        'Comment Tree Working Successfully',
+        async ({ expect, db, generated }) => {
+            const { post1, creator3, comment1, comment2, comment3 } = mustGenerate(generated);
+
+            const commentTree = await getCommentTree(db, post1.id);
+
+            expect(commentTree.length).toStrictEqual(2);
+            expect(commentTree[0].content).toStrictEqual(comment1.content);
+            expect(commentTree[1].content).toStrictEqual(comment2.content);
+            expect(commentTree[0].children?.length).toStrictEqual(0);
+            expect(commentTree[1].children?.length).toStrictEqual(2);
```

Comment:
Would recommend breaking this into two separate checks:
```suggestion
            expect(commentTree[0].children).toBeDefined();
            expect(commentTree[1].children).toBeDefined()
            expect(commentTree[0].children).toHaveLength(0);
            expect(commentTree[1].children).toHaveLength(2);
```